internal/server/middleware/grpc/middleware_test.go

```diff
@@ -694,3 +696,864 @@ func TestCacheUnaryInterceptor_Evaluate(t *testing.T) {
 		})
 	}
 }
+
+func TestAuditUnaryInterceptor_UpdateFlag(t *testing.T) {
```

Comment:
I think we are missing a test for `CreateFlag`

Reply:
🤦  thank you!